### PR TITLE
Finish create ref migration

### PR DESCRIPTION
**Pull request by Markionium**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Finalise the migration to `createRef`

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by Markionium** [comment]
@dzearing Seems like the missing reference checks pushed us over the bundle-size limit

**Comment by Markionium** [comment]
Seems like the removal of @autobind saved enough for this to fit the 📦 size :P

File: packages/experiments/src/components/ExtendedPicker/BaseExtendedPicker.tsx [hunk 1]
```diff
@@ -90,13 +95,13 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>> extend
...
       <div
-        ref={ this._resolveRef('root') }
+        ref={ this.root }
         className={ css(
           'ms-BasePicker',
           className ? className : '') }
       >
         <FocusZone
-          ref={ this._resolveRef('focusZone') }
+          ref={ this.focusZone }
```
Comment (dzearing) [inline packages/experiments/src/components/ExtendedPicker/BaseExtendedPicker.tsx]: should be componentRef

Comment (Markionium) [reply packages/experiments/src/components/ExtendedPicker/BaseExtendedPicker.tsx]: So this is/was a bug?

Comment (Markionium) [reply packages/experiments/src/components/ExtendedPicker/BaseExtendedPicker.tsx]: In that case should this be done everywhere? 

What is the difference in this case between regular ref and componentRef? For example we have this sometimes :P

```ts
<FocusZone
        { ...getNativeProps(this.props, divProperties) }
        direction={ FocusZoneDirection.horizontal }
        ref={ this._onRootRef }
        componentRef={ this._focusZone }
```

**Pull request merged**